whylog/config/parsers.py

```diff
@@ -83,6 +83,18 @@ def convert_params(self, params):
             converted_params.append(converter.convert(params[i]))
         return tuple(converted_params)
 
+    def get_primary_key_group(self):
+        # Here assumption that len of primary_keys_groups equals 1
```

Comment:
I can see that by the assertion below - remove unnecessary comment (before it gets outdated and starts confusing people)